docs/builtin_elements.md

```diff
@@ -638,6 +638,8 @@ When not part of a layout, its width or height defaults to 100% of the parent el
   and a negative value decreases the distance. The default value is 0.
 * **`single-line`** (bool): When set to `true`, no newlines are allowed (default value: `true`)
 * **`wrap`** (*enum [`TextWrap`](#textwrap)*): The way the text input wraps.  Only makes sense when `single-line` is false. (default: no-wrap)
+* **`input-type`** (*enum [`InputType`](#InputType)*): The way to allow special input viewing properties such as password fields
+* **`password-replace-char`** (*string*): The character to use when replacing characters in the input field. The default is "*"
```

Comment:
I'm inclined to say that we could leave the configurability of this one out for now. @ogoffart @hunger what do you think?

Reply:
Yes i agree, this doesn't need to be configurable per widget.

(And it should kind of depend on the style. Qt for example may use \u{2022} or \u{25cf} depending on the platform)

Reply:
Ok. I'll remove the replace-char and references to it